src/cf.data.pre

```diff
@@ -7112,11 +7112,22 @@ LOC: Config.visibleHostname
 DEFAULT: none
 DEFAULT_DOC: Automatically detect the system host name
 DOC_START
-	If you want to present a special hostname in error messages, etc,
-	define this.  Otherwise, the return value of gethostname()
-	will be used. If you have multiple caches in a cluster and
-	get errors about IP-forwarding you must set them to have individual
-	names with this setting.
+	The public domain name (FQDN) of this Proxy.
+
+	This name will be used in generated URLs sent to clients for
+	objects they need to download directly from Squid, and for other
+	generated responses (eg error pages) needing to display the proxy
```

Comment:
s/eg/e.g.,/ to use customary spelling and for consistency sake

Reply:
done

Reply:
You missed a comma after `e.g.` in at least one cases. The rules about following `e.g.` with a comma differ from culture to culture, but we should use a comma for consistency sake, at least when there are multiple examples (although I would just use it everywhere for simplicity).

```
$ fgrep 'e.g.,' -c src/cf.data.pre 
28
$ grep 'e[.]g[.][^,]' -c src/cf.data.pre 
6 # all for single-item examples
```

Besides, using a comma is the more logical than not using it because "for example" is followed by a comma and that is what "e.g." means :\-).